test/integration/testem/testem.js

```diff
@@ -1,16 +1,22 @@
-/* eslint-disable no-global-assign */
 /* eslint-disable import/no-commonjs */
 /* eslint-disable flowtype/require-valid-file-annotation */
-require = require("esm")(module);
-const {generateFixtureJson, getAllFixtureGlobs} = require('./lib/generate-fixture-json');
-const createServer = require('./lib/server');
-const buildTape = require('../../build/test/build-tape');
 const runAll = require('npm-run-all');
 const chokidar = require('chokidar');
 const rollup = require('rollup');
 const notifier = require('node-notifier');
-const rollupDevConfig = require('../../rollup.config').default;
-const rollupTestConfig = require('./rollup.config.test').default;
+
+// hack to be able to import ES modules inside a CommonJS one
+let generateFixtureJson, getAllFixtureGlobs, createServer, buildTape, rollupDevConfig, rollupTestConfig;
+async function loadModules() {
+    const generateFixture = await import('../lib/generate-fixture-json.js');
+    generateFixtureJson = generateFixture.generateFixtureJson;
+    getAllFixtureGlobs = generateFixture.getAllFixtureGlobs;
+
+    createServer = (await import('../lib/server.js')).default;
+    buildTape = (await import('../../../build/test/build-tape.js')).default;
+    rollupDevConfig = (await import('../../../rollup.config.js')).default;
+    rollupTestConfig = (await import('../rollup.config.test.js')).default;
+}
```

Comment:
Minor nit but how about using a `Promise.all([import(), import(),..,])` so all the modules are loaded in parallel?

Reply:
@arindam1993 the code would have to be more verbose (you have to destructure all the `.default` individually), and perf difference is negligible here so I thought it'd be fine as linear.